app/views/newMain.scala.html

```diff
@@ -24,9 +25,48 @@
+				<span class="countryGroups__label">United Kingdom (£)</span>
+				<span class="icon icon--arrows">
+					<svg width="19" height="10" xmlns="http://www.w3.org/2000/svg"><path class="icon__arrow-down" d="M1.25.5l-.75.75L9 9.5h.95l8.55-8.25-.75-.75L9.5 7.25z" /><path class="icon__arrow-up" d="M1.25 9.5L.5 8.75 9 .5h.95l8.55 8.25-.75.75L9.5 2.75z" /></svg>
+				</span>
+			</summary>
+			<ul class="countryGroups__list">
+				<li class="countryGroups__item"><a href="#">
+					<span class="icon">
+						<svg width="18" height="14" xmlns="http://www.w3.org/2000/svg"><path d="M1.36 6.9l-.86.888L4.798 14h.409L17.5.865 16.64 0 5.207 10.694z" /></svg>
+					</span>
+					United Kingdom (£)
+				</a></li>
+				<li class="countryGroups__item"><a href="#">United States (US$)</a></li>
+				<li class="countryGroups__item"><a href="#">Europe (€)</a></li>
+				<li class="countryGroups__item"><a href="#">New Zealand (NZ$)</a></li>
+				<li class="countryGroups__item"><a href="#">Canada (CA$)</a></li>
+				<li class="countryGroups__item"><a href="#">International (US$)</a></li>
+				<li class="countryGroups__item"><a href="#">Australia (AU$)</a></li>
+			</ul>
+		</details>
+	</header>
+	
+	<div class="content">
+		@if(displayForm) {
+			@newPaymentFlow()
+		} else {
+			@newThankYou()
+		}
+	</div>
 
-		@newPaymentFlow()
+	<footer role="contentinfo">
+		<div class="license">
+			<small class="license__txt">© 2018 Guardian News and Media Limited or its affiliated companies. All rights reserved.</small>
 	
+			<p>The ultimate owner of the Guardian is The Scott Trust Limited, whose role it is to secure the editorial and financial independence of the Guardian in perpetuity. Reader contributions support the Guardian’s journalism. Please note that your support of the Guardian’s journalism does not constitute a charitable donation, as such your contribution is not eligible for Gift Aid in the UK nor a tax-deduction elsewhere. If you have any questions about contributing to the Guardian, please <a href="mailto:contribution.support@@theguardian.com">contact us here</a>.</p>
```

Comment:
Double @@, should be `<a href="mailto:contribution.support@theguardian.com">`?

For a future PR - the contact email is different for Australia, [defined here](https://github.com/guardian/support-frontend/blob/master/assets/helpers/legal.js#L32) .

Reply:
Very good, thanks! Added to [the card](https://trello.com/c/zRSxRltu/789-npf-class-names-bem-data-in-currencyjs-contributionsjs-objectkeys)

Reply:
To @tsop14's point, I think this is handled by the `ContribLegal` component in React-land, it takes the country group and applies the correct email address.